weave

```diff
@@ -804,6 +816,51 @@ dns_args() {
     fi
 }
 
+default_etc_hosts() {
+    name=$1
+    shift
+    extra_hosts=$@
+    echo "# created by Weave - BEGIN"
+    echo "# container hostname"
+    for cidr in $ALL_CIDRS ; do
+        echo "${cidr%/*}    $name"
+    done
+    echo
+    echo "# static names added with --add-host"
+    for host in $extra_hosts ; do
+        name=$(echo $host | cut -f1 -d":")
+        ip=$(echo $host | cut -f2 -d":")
+        echo "$ip     $name"
+    done
+
+    cat <<-EOF
+
+# default localhost entries
+127.0.0.1       localhost
+::1             ip6-localhost ip6-loopback
+fe00::0         ip6-localnet
+ff00::0         ip6-mcastprefix
+ff02::1         ip6-allnodes
+ff02::2         ip6-allrouters
```

Comment:
This is copy&paste of the standard entries in Ubuntu systems... I think it is safe to leave it as it is...